numba/_lapack.c

```diff
@@ -466,6 +514,80 @@ typedef void (*cgelsd_t)(F_INT *m, F_INT *n, F_INT *nrhs, void *a, F_INT *lda,
                          F_INT *info);
 
 
+
+/*
+ * kind_size()
+ * gets the data size appropriate for a specified kind.
+ *
+ * Input:
+ * kind - the kind, one of:
+ *         (s, d, c, z) = (float, double, complex, double complex).
+ *
+ * data_size - modified in place, on return contains the appropriate data size.
+ *
+ * Returns 0 on success, -1 else.
```

Comment:
heh, that's not a new change, its a wholesale move of a previously reviewed code block! But yes, not quite!